daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md

```diff
@@ -69,6 +72,30 @@ curl -X POST http://localhost:3500/v1.0/bindings/myRabbitMQ \
       }'
 ```
 
+## Specifying a priority on message level
+
+Priority can be defined at the message level. If `maxPriority` parameter is set, high priority messages will have priority over other low priority messages. 
+
+To set priority at message level use the `metadata` section in the request body during the binding invocation.
+
+The field name is `priority`.
+
+Example:
+
+```shell
+curl -X POST http://localhost:3500/v1.0/bindings/myRabbitMQ \
+  -H "Content-Type: application/json" \
+  -d '{
+        "data": {
+          "message": "Hi"
+        },
+        "metadata": {
+          "priority": "5"
```

Comment:
The priority field is not described in the metadata parameters above (only maxPriority is). Can you add priority to this also, with a description of how/when to use this? 

Also what the relationship between setting priority and maxPriority on the binding?

Reply:
The priority field is correctly shown here as metadata you give the API. It's for the publisher to set the priority of the message and it doesn't appear on the RabbitMQ component.